packages/frontend/web/browser/sentry/sentry.tsx

```diff
@@ -0,0 +1,62 @@
+import * as Sentry from '@sentry/browser';
+
+// This is set to 10% here rather than the normal 1% so we get good
+// feedback early on. TODO: This should be reduced to 1% once the site is fully deployed.
+const SAMPLE_RATE = 0.1;
+
+// Only send errors matching these regexes
+const whitelistUrls = [
+    /webpack-internal/,
+    /localhost/,
+    /assets\.guim\.co\.uk/,
+    /ophan\.co\.uk/,
+];
+
+// Ignore these errors
+const ignoreErrors = [
+    // https://docs.sentry.io/platforms/javascript/#decluttering-sentry
+    "Can't execute code from a freed script",
+    /InvalidStateError/gi,
+    /Fetch error:/gi,
+    'Network request failed',
+    'This video is no longer available.',
+    'UnknownError',
+];
+
+export const initialiseSentry = (adBlockInUse: boolean) => {
+    const { editionLongForm, contentType } = window.guardian.app.data.CAPI;
+    const {
+        isDev,
+        switches: { enableSentryReporting },
+    } = window.guardian.app.data.config;
+
+    Sentry.init({
+        ignoreErrors,
+        whitelistUrls,
+        dsn: 'https://1937ab71c8804b2b8438178dfdd6468f@sentry.io/1377847',
```

Comment:
Should this be hardcoded? I think [previously](https://github.com/guardian/dotcom-rendering/pull/831/files#diff-85c4844add4bd4441488aec0fff17145L78-L80) it was coming from data on the `window`

Reply:
This is now done.